eng/ReferenceAssemblies.props

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <IsReferenceAssembly Condition="'$(IsReferenceAssembly)'=='' AND ($(MSBuildProjectFullPath.Contains('\ref\')) OR $(MSBuildProjectFullPath.Contains('/ref/')))">true</IsReferenceAssembly>
```

Comment:
This isn't used in any props declared/imported before this file gets pulled in, it is only used by targets, however, we could potentially move this declaration to Directory.Build.props root if we want all this properties to be declared at the root of the stack